widgetastic_manageiq.py

```diff
@@ -842,16 +843,16 @@ def _parse_pages(self):
         # obtaining amount of existing pages, there is 1 page by default
         if item_amt == 0:
             page_amt = 1
-        elif item_amt % items_per_page != 0:
-            page_amt = item_amt // items_per_page + 1
         else:
-            page_amt = item_amt // items_per_page
+            # round up after dividing total item count by per-page
+            page_amt = int(ceil(float(item_amt) / float(items_per_page)))
```

Comment:
looks more gracefully :+1: . I thought of using math but didn't want to add additional dependencies.